lib/linting/schemas/definitions/tags.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
     'stylus support': 'For content related to stylus input integration and related stylus interactions',
     'touchscreen support': 'For content related to checking touchscreen availability on devices and handling touchscreen events when appropriate',
     'trackpad support': 'For content related to handling trackpad-specific interactions and gestures',
+    'camera orientation': 'For content related to handling orientations in an Android app with camera functionality',
```

Comment:
```suggestion
    'camera': 'For content related to handling camera functionality',
```

I think a more generic `camera` tag here would catch more content, and therefore build a more robust collection, than the specific `camera orientation`

Reply:
Thanks for your suggestion!
Is there anyway we can keep "orientation"? I added "large screen" because this is somewhat related (small screens, phones, don't have orientation issues).
This is an important keyword for the article, and I think it makes sense to include it to make the content more discoverable.

Reply:
To be more specific, I'm thinking if we can have 2 tags: "camera" and "orientation". What do you think?

Reply:
I'm open to the idea, but let's. tease it out a little more. There are two different orientations we could be talking about; device orientation and window orientation, which. are somewhat related but independent. Are you recommending one "orientation" tag to cover both, separate tags, or something else?

Reply:
That's a very good question. I think this article is relevant to both.

Hmm... I was going to suggest "orientation" because this can also be added to window-management.
However I see that we already have a "device configuration" tag
> 'device configuration': 'For content related to the device mode (clamshell, tablet, touchscreen), orientation (portrait, orientation), and other configuration changes'
This seems relevant to this article. Can I use "camera" and "device configuration" here?

Reply:
Yah, I think camera and device configuration are good tags for this.